Rakefile

```diff
@@ -42,3 +48,58 @@ end}.strip
     exit # otherwise rake will try to run the other arguments
   end
 end
+
+desc "Generate the weekly winners playlist"
+task :create_playlist do
+  Playlist.weekly_winners
+end
+
+desc "Wipe users from database"
+task :db_delete_users do
+  User.delete_all
+end
+
+desc "Wipe playlists from database"
+task :db_delete_playlists do
+  Playlist.delete_all
+  PlaylistSong.delete_all
+end
+
+desc "Wipe votes from database"
+task :db_delete_votes do
+  Vote.delete_all
+end
+
+desc "Wipe songs from database"
+task :db_delete_songs do
+  Song.delete_all
+end
+
+desc "Create playlist in week n."
+task :create_playlist do
+  require './create_playlist.rb'
+end
+
+desc "Create votes in week n."
+task :create_votes do
+  require './create_votes.rb'
+end
+
+desc "Create users"
+task :create_users do
+  require './create_users.rb'
+end
+
+desc "Create songs in week n"
+task :create_songs do
+  require './create_songs.rb'
+end
+
+desc "Wipe database records"
+task :delete_db do
+  User.delete_all
+  Playlist.delete_all
+  PlaylistSong.delete_all
+  Vote.delete_all
+  Song.delete_all
+end
```

Comment:
I like these in general, but it makes me a tad nervous that someone might accidentally `heroku run` something they shouldn't. 💡 : check `Rails.env.production?` before doing things like this.